hasura/graphql-engine
# console: fix import metadata button text (close #5386)

### Description



### Changelog


### Affected components



### Related Issues



https://github.com/hasura/graphql-engine/issues/5386

### Solution and Design


It would be more appropriate for the state to change to `Importing...` after the file is selected.
I changed setState timing.

- Open file upload window (`isImporting: false`)
- Select a file and click OK (`isImporting: true`)
- In both cases of success and failure, done uploadFile (`isImporting: false`)

### Steps to test and verify


verify cancel state
1. Open the console and visit `settings/metadata-actions`
2. Click `Import metadata` button
3. Click `cancel`

verify upload state
1. Open the console and visit `settings/metadata-actions`
2. Click `Import metadata` button
3. Select file and upload it
